.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,59 @@
+name: Lint and Test
+
+on:
+    workflow_call:
+
+jobs:
+
+    run_linkml_tasks:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+
+        steps:
+            - name: Repo checkout
+              uses: actions/checkout@main
+
+            - name: Set up Python 3.
+              uses: actions/setup-python@main
+              with:
+                python-version: 3.12
+
+            - name: Install Poetry and dependencies.
+              run: |
+                pipx install poetry
+                poetry install
+
+            - name: Lint linkml file
+              id: lint_linkml
+              run: |
+                  uv run linkml-lint ./src/bertron_schema/schema/bertron_schema.yaml --ignore-warnings
+              continue-on-error: true
+
+            - name: Validate sample data against the schema
+              id: validate_sample_data
+              run: |
+                  uv run linkml-validate -s ./src/bertron_schema/schema/bertron_schema.yaml ./src/data/examples/valid/*.json
+              continue-on-error: true
+
+            - name: Test documentation generation
+              id: test_docgen
+              run: |
+                poetry install --with docs
+                mkdir -p docs
+                make gendoc
```

Comment:
generating docs brings up errors that the linkml linter doesn't catch, so adding in a test that the docs can be created (they do not get deployed if they are successfully generated).